src/OrdersList/hooks/useSearchableIndexes/useSearchableIndexes.js

```diff
@@ -1,9 +1,42 @@
 import { useMemo } from 'react';
+import { useIntl } from 'react-intl';
 
 import { useLocaleDateFormat } from '@folio/stripes-acq-components';
 
-export function useSearchableIndexes() {
+import { FILTERS } from '../../constants';
+
+export function useSearchableIndexes(customFields) {
   const localeDateFormat = useLocaleDateFormat();
+  const intl = useIntl();
+
+  const customFieldsSearchableIndexes = useMemo(() => {
+    let result = [];
+
+    if (customFields) {
+      result = customFields.map(cf => {
+        const customFieldLabel = intl.formatMessage({ id: 'stripes-smart-components.customFields' });
+        const fieldLabel = `${customFieldLabel} ${cf.name}`;
+        const fieldValue = `${FILTERS.CUSTOM_FIELDS}.${cf.refId}`;
+
+        if (cf.type === 'TEXTBOX_LONG' || cf.type === 'TEXTBOX_SHORT') {
+          return {
+            label: fieldLabel,
+            value: fieldValue,
+          };
+        } else if (cf.type === 'DATE_PICKER') {
```

Comment:
The same comment https://github.com/folio-org/ui-orders/pull/1564/files#r1513982018